### DISCUSSION TITLE
Mark to never delete

### DISCUSSION THREAD

**Discussion by cbarth3** [lavinir/hassio-onedrive-backup#128] [Ideas]
I would like to be able to have the option to tell the addon not to delete a certain backup. I generally like to be able to keep the first backup of the month just as a "just in case" I am moving over to this onedrive backup addon from the google drive one that the below screenshot is from.

![image](https://github.com/lavinir/hassio-onedrive-backup/assets/27602300/1c57a8f0-b9be-439a-a382-b282de2dc167)

**Comment by lavinir** [comment]
I will add this to the backlog. For now you can make use of the generational backups to get close something close.

**Comment by lavinir** [comment]
This is now available in 2.3
